src/app/curated-rolls/curated-roll-reader.ts

```diff
@@ -66,8 +73,5 @@ function toDimWishListCuratedRoll(textLine: string): CuratedRoll | null {
 export function toCuratedRolls(bansheeText: string): CuratedRoll[] {
   const textArray = bansheeText.split('\n');
 
-  return _.uniqWith(
-    _.compact(textArray.map((line) => toCuratedRoll(line) || toDimWishListCuratedRoll(line))),
-    deepEqual
-  );
+  return _.compact(textArray.map((line) => toDimWishListCuratedRoll(line) || toCuratedRoll(line)));
```

Comment:
This is the bit that worries me - there's a lot of duplicate lines in wish lists (different people tend to recommend the same rolls), so this was done in to try to tease a `Set` out of the data provided.

Nothing will break if we have dupe lines, but it becomes more data to hang on to (and the `nnnn rolls loaded` message we give after the load finishes will be a bit misleading).

Reply:
Hmm, that's tough. I suspect that the `uniqWith` is a big slowdown, since it's implemented as a O(n^2) number of comparisons. (Plus, comparing two sets is *also* O(n^2), so each comparison also involves an expensive comparison, and `deepEquals` isn't exactly fast).

Reply:
On my machine it takes 62ms to load the rolls and 15.2 *seconds* to uniq them.